internal/binutils/binutils_test.go

```diff
@@ -401,3 +414,125 @@ func TestOpenMalformedMachO(t *testing.T) {
 		t.Errorf("Open: got %v, want error containing 'Mach-O'", err)
 	}
 }
+
+func TestObjdumpVersionChecks(t *testing.T) {
+	// Test that the objdump version strings are parsed properly.
+	type objdumpTest struct {
+		desc string
+		os   string
+		ver  string
+		want bool
+	}
+
+	for _, tc := range []objdumpTest{
+		{
+			// Valid Apple LLVM version string with usable version.
+			desc: "Valid Apple LLVM version string",
+			os:   "darwin",
+			ver: `Apple LLVM version 11.0.3 (clang-1103.0.32.62)
+			Optimized build.`,
+			want: true,
+		},
+		{
+			// Valid Apple LLVM version string with unusable version.
+			desc: "Unusable Apple LLVM version",
+			os:   "darwin",
+			ver: `Apple LLVM version 10.0.0 (clang-1000.11.45.5)
+			Optimized build.`,
```

Comment:
I think using usual string instead of backtick string would be more readable: `"Apple LLVM version 10.0.0 (clang-1000.11.45.5)\nOptimized build."`

It would also more precisely match the output of the tool - with backtick strings there is some leading whitespace on 2+ lines due to the indentation.

Similar below.

Reply:
Done. Exceeding 80 columns in some cases as a result.